scanpy/tools/_ingest.py

```diff
@@ -26,10 +30,21 @@ def ingest(
     """
     ing = Ingest(adata_ref)
     ing.transform(adata)
-    ing.map_embedding(embedding_method)
+    embedding_method = _str_to_list(embedding_method)
+    for method in embedding_method:
+        ing.map_embedding(method)
     if obs is not None:
         ing.neighbors(**kwargs)
-        ing.map_labels(obs, labeling_method)
+
+        obs = _str_to_list(obs)
+        labeling_method = _str_to_list(labeling_method)
```

Comment:
Just a few style thoughts.

* I think `_str_to_list` may be [simple enough it doesn't need a function](http://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection).
* Could the preparing of `obs` and `labeling_method` (via `_str_to_list`) happen at the top of the function? I don't think it would break anything, and I find code easier to follow when a variables value is consistent through as much of the function as possible.

Reply:
I'll fix it.